Monika After Story/game/script-topics.rpy

```diff
@@ -15413,3 +15413,47 @@ label monika_why_do_you_read:
     extend 3eua "While my situation in here has its downsides, [line_mid] opportunity to spend more time on the things I like."
     m 1ekbsu "...Though then again, nothing could ever beat spending more time with you~"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_staring",
+            category=['monika, romance'],
+            prompt="Staring contest",
+            random=True,
+            aff_range=(mas_aff.ENAMORED, None)
+
+        )
+    )
+
+label monika_staring:
+    if not seen_event("monika_staring"):
```

Comment:
My bad, I think this check will always return `True`, let's check `renpy.seen_label`
```suggestion
    if not renpy.seen_label("monika_staring"):
```